### ISSUE TITLE
inspecting elements not working properly

### ISSUE THREAD

**Issue by raphaelquast** [spyder-ide/spyder#11927]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description

inspect element (e.g. `crtl + i`) is showing some very strange behaviour...

### What steps reproduce the problem?

1. type something like the following in the editor **(do NOT use tab - autocompletion!)**

``` python
import numpy as np
np.cos
```
and then hit `crtl + i` to inspect `np.cos` from the editor

``` python
>>> No documentation available
```
is shown in the `Help` panel (and the Object-field on top of the help-panel is empty)
(also if the first line is executed)

2. do the same in the **console**
--> still no documentation for `np.cos` but now the Object-field on top of the help-panel shows `cos`   --> change `cos`  to  `np.cos`  in the Object-field   --> the documentation appears!

3. now type `np.array` instead of `np.cos` in the editor and hit `crtl + i`   --> the documentation appears!

4.  do the same again in the **console**
--> now the Object-field on top of the help-panel shows `array`  and the documentation for the array-module appears...  --> change `array`  to  `np.array`  in the Object-field   --> the correct docstring of the numpy-array function appears!

### What is the expected output? What do you see instead?

I'd expect to see the docstring of the inspected object :-)

## Versions

* Spyder version: 4.1.1
* Python version: 3.6.7
* Qt version: 5.9.6
* PyQt version: 5.9.2
* Operating System name/version: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.3.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.2.2 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.13.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.2 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=0.25                  :  2.4.4 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.9 (OK)
qdarkstyle >=2.8               :  2.8 (OK)
qtawesome >=0.5.7              :  0.7.0 (OK)
qtconsole >=4.6.0              :  4.7.1 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.3 (OK)
sphinx >=0.6.6                 :  2.4.0 (OK)
spyder_kernels >=1.9.0;<1.10.0 :  1.9.0 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  18.1.1 (OK)

# Optional:
cython >=0.21                  :  None (OK)
matplotlib >=2.0.0             :  3.1.3 (OK)
numpy >=1.7                    :  1.17.2 (OK)
pandas >=0.13.1                :  0.25.1 (OK)
scipy >=0.17.0                 :  1.3.1 (OK)
sympy >=0.7.3                  :  1.4 (OK)

```

**Comment by ccordoba12** [comment]
Duplicate of #11821

**Comment by ccordoba12** [comment]
Thanks for taking the time to report it in such detail. We'll try to fix this in 4.1.2 or 4.1.3

**Comment by raphaelquast** [comment]
OK perfect :-) Thanks for the response!
